docs/content/platform/blocks/basic.md

```
@@ -1,83 +1,290 @@
 ### What it is
-A basic block that prints text to the console for debugging purposes.
+Block for file upload input (string path for example).
 
 ### What it does
-This block takes a text input and prints it to the console, then outputs a status message.
+Block for file upload input (string path for example).
 
 ### How it works
-It receives a text input, prints it to the console with a "Print: " prefix, and then yields a "printed" status.
+<!-- MANUAL: how_it_works -->
+This block provides a file upload input for users interacting with your agent. Users can upload files which are stored temporarily and passed to your workflow as a file path reference.
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)